modules/installation-initializing.adoc

```diff
@@ -362,6 +363,36 @@ endif::[]
 ifndef::restricted[]
 . Modify the `install-config.yaml` file. You can find more information about
 the available parameters in the *Installation configuration parameters* section.
```

Comment:
Double-quotations for section titles (per IBM SG).

Reply:
Fixed.